common/cassandra/cassandraCluster.go

```diff
@@ -60,26 +61,57 @@ func NewCassandraCluster(cfg config.Cassandra) (*gocql.ClusterConfig, error) {
 		cluster.HostFilter = gocql.DataCentreHostFilter(cfg.Datacenter)
 	}
 	if cfg.TLS != nil && cfg.TLS.Enabled {
+		if cfg.TLS.CertData != "" && cfg.TLS.CertFile != "" {
+			return nil, errors.New("Cannot specify both certData and certFile properties")
+		}
+
+		if cfg.TLS.KeyData != "" && cfg.TLS.KeyFile != "" {
+			return nil, errors.New("Cannot specify both keyData and keyFile properties")
+		}
+
+		if cfg.TLS.CaData != "" && cfg.TLS.CaFile != "" {
+			return nil, errors.New("Cannot specify both caData and caFile properties")
```

Comment:
Maybe this should be additive though? Because users may want multiple CA with some being intermediate and some specified in files with others in bytes?

Reply:
talked offline. we'll keep it simple for now (also removed merging behavior for the regular TLS specifications), but if there is user interest, then we can add in merging fairly easily